connexion/operation.py

```diff
@@ -264,16 +264,17 @@ def _retrieve_reference(self, reference):
         return definition
 
     def get_mimetype(self):
+        # if the endpoint as no 'produces' then the default is 'application/json'
+        default_mimetype = 'application/json'
```

Comment:
As this is a constant you should put it a the module level to avoid creating a string every time get_mimetype is called.